test/accessor_tests.cc

```diff
@@ -650,16 +650,20 @@ namespace detail {
 	template <int>
 	class acc_out_of_bounds_kernel {};
 
-	TEMPLATE_TEST_CASE_METHOD_SIG(oob_fixture, "accessor reports out-of-bounds accesses", "[accessor][oob]", ((int Dims), Dims), 1, 2, 3) {
+	TEMPLATE_TEST_CASE_METHOD_SIG(oob_fixture, "device accessor reports out-of-bounds accesses", "[accessor][oob]", ((int Dims), Dims), 1, 2, 3) {
 #if !CELERITY_ACCESSOR_BOUNDARY_CHECK
 		SKIP("CELERITY_ACCESSOR_BOUNDARY_CHECK=0");
 #endif
-		buffer<int, Dims> buff(range_cast<Dims>(range<3>{10, 20, 30}));
+		buffer<int, Dims> buff_nl(range_cast<Dims>(range<3>{10, 20, 30}));
+		buffer<int, Dims> buff_nf(range_cast<Dims>(range<3>{10, 20, 30}));
```

Comment:
From what I gather these variants are for unnamed/named, but I cannot figure out what the "nl" and "nf" abbreviations stand for..? 🤔